content/DC_resistivity/images/.picasa.ini

```diff
@@ -0,0 +1,5 @@
+[zongtx.gif]
+faces=rect64(78518b1aa2fcd012),ffffffffffffffff
+backuphash=52491
+[ohm_mapper.jpg]
+backuphash=64278

```

Comment:
This file doesn't need to be in the repo. Since the .ini was added to the gitignore after committing this. You can take care of this using the command line: `git rm --cached /\*.ini` which will remove all of the .ini files from the repo, but leave your local copies of them (http://stackoverflow.com/questions/6535362/gitignore-after-commit)